students/jarosz_natalia/lesson_01/First digit after decimal point.py

```diff
@@ -1,5 +1,4 @@
-
 a = float(input())
 w1 = a // 0.1
 w2 = w1 % 10
-print(round(w2))
\ No newline at end of file
+print(round(w2))
```

Comment:
I have no idea why this solution doesn't work :( Could you explain it to me

Reply:
That's because of the usual floating point rounding issues. Everything is going to be wrong from the second line (w1 = a // 0.1),  where for the input value 12.9 you got in a result w1 = 128.0 so it differs from the result you expect - it's not your fault. The whole algorithm to calculate this task is proper. Your code passed all snakify tests. One of the ways to resolve this task is: 
a = float(input())
print(int((value - int(value)) * 10))

I think that we can proceed with the merge.